modules/dnn/test/test_tflite_importer.cpp

```diff
@@ -210,6 +210,9 @@ TEST_P(Test_TFLite, max_unpooling)
 }
 
 TEST_P(Test_TFLite, EfficientDet_int8) {
+    if (backend == DNN_BACKEND_INFERENCE_ENGINE_NGRAPH)
+        applyTestTag(CV_TEST_TAG_DNN_SKIP_IE_NGRAPH); // TODO: fix this test for OpenVINO
```

Comment:
Did test work before the patch?

Reply:
This test doesn't work in both 4.x and 5.x branches. Probably because I use the newest OpenVINO version

Reply:
Test definitely worked on 2023 release of OpenVINO. I will check 2024 version. I think ok to skip on this PR.